c/include/ml-api-common.h

```diff
@@ -142,6 +142,13 @@ typedef enum _ml_tensor_type_e
   ML_TENSOR_TYPE_UNKNOWN         /**< Unknown type */
 } ml_tensor_type_e;
 
+/**
+ * @brief The function to be called when destroying the data using in machine learning API.
+ * @since_tizen 7.0
+ * @param[in] user_data The data to be passed to the callback function.
```

Comment:
that phrase is used in when the user gives user_data to when he sets what it will be, here you can use "The user data passed from the callback registration function" or similar

Reply:
Thanks, your suggestion is great, and I followed it.